docs/docs/_available-tags.md

```diff
@@ -0,0 +1,24 @@
+# Available tags
+
+The following is a list of available tags that can be used when contributing a sample. Click on the links to view the samples associated with the tags.
+
+- [AI](./tags#AI)
+- [Analytics](./tags#Analytics)
+- [Cleanup](./tags#Cleanup)
+- [Customizations](./tags#Customizations)
+- [Files](./tags#Files)
+- [Governance](./tags#Governance)
+- [Guests](./tags#Guests)
+- [Lists](./tags#Lists)
+- [Migration](./tags#Migration)
+- [Miscelaneous](./tags#Miscelaneous)
```

Comment:
Looks like a typo here

```suggestion
- [Miscellaneous](./tags#Miscellaneous)
```

Reply:
This also needs to be updated in all samples

Reply:
Yes,  but I'll remove the miscellaneous tag anyway